Description: Fix FTBFS with gnupg 1.4.12
 .
 gnupg 1.4.12 adds a new status information DECRYPTION_INFO. Add check
 for this new status message.
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=76173
Bug-Debian: http://bugs.debian.org/665226
Forwarded: yes
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2012-04-14

--- a/GnuPG.pm
+++ b/GnuPG.pm
@@ -759,6 +759,8 @@
     } else {
         # gnupg 1.0.2 adds this status message
         ( $cmd, $arg ) = $self->read_from_status() if $cmd =~ /BEGIN_DECRYPTION/;
+        # gnupg 1.4.12 adds this status message
+        ( $cmd, $arg ) = $self->read_from_status() if $cmd =~ /DECRYPTION_INFO/;
 
         $self->abort_gnupg( "invalid passphrase - $cmd" ) unless $cmd =~ /PLAINTEXT/;
     }
